src/lib.rs

```diff
@@ -607,17 +608,46 @@ impl StructuredExtendedInformation {
         13 => deprecates_fpu_cs_ds,
         // 14 - reserved
         15 => pqe,
-        // 16-17 - reserved
+        16 => avx512f,
+        17 => avx512dq,
         18 => rdseed,
         19 => adx,
         20 => smap,
-        // 21-24 - reserved
-        25 => intel_processor_trace
-        // 26-31 - reserved
+        21 => avx512ifma,
+        // 22-24 - TODO
+        25 => intel_processor_trace,
+        26 => avx512pf,
+        27 => avx512er,
```

Comment:
I don't understand where you are getting these names:

> Bits 27 - 26: Reserved.

Reply:
`avx512pf` and `avx512er` are in [the Intrinsics Guide]( https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=avx512er&expand=4104,3261,3264,3275,3278,3276,3273,3270,3267,3262,4104,2177,2176) and [The Manual (October 2017)](https://software.intel.com/sites/default/files/managed/39/c5/325462-sdm-vol-1-2abcd-3abcd.pdf).